src/bin/bat/app.rs

```diff
@@ -169,9 +169,7 @@ impl App {
                 || match self.matches.value_of("color") {
                     Some("always") => true,
                     Some("never") => false,
-                    Some("auto") | _ => {
-                        env::var_os("NO_COLOR").is_none() && self.interactive_output
-                    }
+                    _ => env::var_os("NO_COLOR").is_none() && self.interactive_output,
```

Comment:
Honestly, I prefer the other way. By removing the `Some("auto")`, it no longer explicitly covers every possible option.

It may be redundant, but it's more clear.

Reply:
There are different solutions:
- say it with a comment, something like:
  ```
                      _ => {
                        // this case includes also Some("auto")
                        env::var_os("NO_COLOR").is_none() && self.interactive_output
                    }
  ```
- throw an error:
```
                    Some("auto") => env::var_os("NO_COLOR").is_none() && self.interactive_output,
                    _ => return Err("unvalid color option".into()),
```
- disable local clippy warning and revert the change:
```
    #[allow(clippy::clippy::wildcard_in_or_patterns)]
    pub fn config(&self, inputs: &[Input]) -> Result<Config> {
```

Reply:
If all cases are definitely covered otherwise, we can also do
```
Some("auto") => …
_ => unreachable!("…")
```

Reply:
Here we do something very similar:
https://github.com/sharkdp/bat/blob/2b82203041e0a4242659b95a4fa2ae6454497637/src/bin/bat/app.rs#L105
If you prefer we can do something like:

```rust
Some("auto") => …
_ => unreachable!("other values for --color are not allowed")
```

Reply:
I would prefer that. I added a commit